src/components/AddressLookup/index.tsx

```diff
@@ -112,7 +127,11 @@ const AddressLookup: FunctionComponent<AddressLookupProps> = ({
 
   const search = (query: string, id = '') => {
     if (query.length === 0) return setAddressOptions([]);
-    fetch(`${FIND_URL}?Key=${apiKey}&Text=${query}&Container=${id}`)
+    let searchUrl = `${FIND_URL}?Key=${apiKey}&Text=${query}&Container=${id}`;
+    if (countries) {
+      searchUrl = `${searchUrl}&Countries=${countries}`;
+    }
```

Comment:
`const searchUrl = `${FIND_URL}?Key=${apiKey}&Text=${query}&Container=${id}${countries ? `&Countries=${countries}` : ''}`;`

if we want to do this without mutations